scripts/shell/run.sh

```diff
@@ -49,7 +49,7 @@ then
     case $format in
     PMML )
         java -Dconfig.file=/opt/openscoring/application.conf -jar /opt/openscoring/openscoring-server-executable-2.0.1.jar --port 8080 &
-        sleep 10s;
+        sleep 15s;
```

Comment:
Why do we need sleep here?

Reply:
Because it start a java process in the same container, need wait openscoring server running.

Reply:
In Clever 1.6.0, using a pod to run openscoring server, mode analyzer wait pod running, and then it try three times, sleep 3s for each loop.